policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -651,9 +652,11 @@ impl Index {
         }
     }
 
-    fn reinitialize_egress_watches(&mut self) {
+    fn reinitialize_egress_watches(&mut self, namespace: Arc<String>) {
```

Comment:
can take a &str instead of an Arc<String> here to avoid needing to clone at the callsite.